packages/tasks/src/model-libraries.ts

```diff
@@ -165,6 +165,14 @@ export const MODEL_LIBRARIES_UI_ELEMENTS = {
 		filter: false,
 		countDownloads: `path_extension:"pth"`,
 	},
+	"depth-pro": {
+		prettyLabel: "DepthPro",
+		repoName: "Depth Pro",
```

Comment:
A particular reason why having `DepthPro` and `Depth Pro`?

Reply:
Btw, when reviewing these PRs I always wonder if we should keep only 1 field given how similar they are

Reply:
The official name seems to be "Depth Pro", as that's what appears in the GitHub repo. I used `DepthPro` for the label because in most other cases (but not all) we use single words. Happy to change to `Depth Pro`!